functionary/core/api/v1/views/task.py

```diff
@@ -67,8 +80,22 @@ def get_serializer_class(self):
         },
         parameters=HEADER_PARAMETERS,
     )
-    def create(self, request, *args, **kwargs):
-        request_serializer = self.get_serializer(data=request.data)
+    def create(self, request: Request, *args, **kwargs):
+        if not request.data.get("parameters"):
```

Comment:
This changes parameters from a required to an optional field, which may or may not have been the intent, but either way that change is not reflected in the endpoint documentation.  I'd personally just keep it as required for the sake of implementation simplicity and explicitness in execution.

Reply:
Fixed. The original intention was that if a user passed a file parameter to a function that only accepts a file parameter, then the parameter field would need to be created if it did not already exist